src/resources/app/components/ListComment.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-card 
+    flat
+  >
+    <div class="d-flex">
+      <v-avatar class="bg-primary mr-4" id="avatar" size="40"></v-avatar>
+      <div>
+        <div id="author">
+          {{ name }}
+        </div>
+        <div id="date" class="text-sm text-textGray">
+          {{ commentDate }}
+        </div>
+      </div>
+    </div>
+    <div id="content" class="ms-14 mt-1">
+      {{ content }}
+    </div>
+    <v-divider id="divider" color="#CECECE" class="my-2"></v-divider>
+  </v-card>
+</template>
+
+<script>
+export default {
+  name: "ListComment",
+  props: [
+    "name",
+    "date",
+    "content",
+  ],
+  data() {
+    return {
+      commentDate: this.dateSlicing(),
+    }
+  },
+  methods: {
+    dateSlicing() {
```

Comment:
please check on `src/resources/api/utils/dateSlicing.js`. you could utilize that function instead of creating a new one.